src/main/java/club/sk1er/patcher/mixins/bugfixes/NetHandlerPlayClient_FixHandleSpawnPlayerNPE.java

```diff
@@ -0,0 +1,50 @@
+package club.sk1er.patcher.mixins.bugfixes;
+
+import net.minecraft.client.network.NetHandlerPlayClient;
+import net.minecraft.client.network.NetworkPlayerInfo;
+//#if MC==10809
+import net.minecraft.network.play.server.S0CPacketSpawnPlayer;
```

Comment:
1. Imports with preprocessor statements should be placed below regular imports.
2. If you add `net.minecraft.network.play.server.SPacketSpawnPlayer net.minecraft.network.play.server.S0CPacketSpawnPlayer` to `versions/1.12.2-1.8.9.txt`, preprocessor isn't needed in this file. Put this line with the rest of the packet mappings.

Reply:
it's done